README.md

```diff
@@ -14,22 +14,23 @@ npm install oas-normalize --save
 
 # Usage
 
-It's pretty simple:
-
 ```javascript
-const OASNormalize = require('oas-normalize');
+import OASNormalize from 'oas-normalize';
```

Comment:
Wanna include several `import`/`require` examples for importing in TS, CJS, ESM, etc?